src/lib/utils/index.ts

```diff
@@ -31,6 +31,12 @@ export { default as mergeSearchParameters } from './mergeSearchParameters';
 export { default as resolveSearchParameters } from './resolveSearchParameters';
 export { default as toArray } from './toArray';
 export { warning, deprecate } from './logger';
+export {
+  default as escapeHits,
```

Comment:
```suggestion
  escapeHits,
```

we should make it a regular export, not a default one, since we're touching the file anyway now

Reply:
Good idea! All done in https://github.com/algolia/instantsearch.js/pull/4618/commits/89280b5ba96293392bd5ee4ed6268ac5c6b04a0c